__tests__/app.test.js

```diff
@@ -279,6 +278,34 @@ describe("/api/articles", () => {
 	});
 });
 
+describe('/api/comments', () => {
+	describe('DELETE /api/comments/:comment_id', () => { 
+		test('STATUS 204: Deletes comment by comment_id', () => {
+			return request(app)
+			.delete("/api/comments/1")
+			.expect(204)
+		})
+		test('STATUS 404: Responds with error if no such comment_id exists', () => {
+			return request(app)
+			.delete("/api/comments/899456")
```

Comment:
Be wary of such large ids, I've seen errors because of it. In our test data its tiny, a solid 99 would do.